site/src/testHelpers/entities.ts

```diff
@@ -2752,3 +2752,13 @@ export const MockListeningPortsResponse: TypesGen.WorkspaceAgentListeningPortsRe
       { process_name: "", network: "", port: 8081 },
     ],
   };
+
+export const DeploymentHealthUnhealthy = {
```

Comment:
Should this have some kind of type annotation (even if it's just through `satisfies`)?